index.template.5.php

```diff
@@ -0,0 +1,127 @@
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/head-2.html"); ?>
+	<!-- TemplateBeginEditable name="head" -->
+	<!-- Place optional header elements here -->
+	<!-- TemplateEndEditable -->
+	<!-- TemplateParam name="class" type="text" value="" -->
+</head>
+<body class="@@(_document['class'])@@ unl" data-version="5.0">
+<?php wdnInclude("/wdn/templates_5.0/includes/global/skip-nav.html"); ?>
+<header class="dcf-header" id="dcf-header" role="banner">
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/header-global-1.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/nav-global-1.html"); ?>
+				<?php wdnInclude("/wdn/templates_5.0/includes/global/visit-global-1.html"); ?>
+				<!-- TemplateBeginEditable name="visitlocal" -->
+				<?php wdnInclude("/wdn/templates_5.0/includes/local/visit-local.html"); ?>
+				<!-- TemplateEndEditable -->
+				<?php wdnInclude("/wdn/templates_5.0/includes/global/visit-global-2.html"); ?>
+				<?php wdnInclude("/wdn/templates_5.0/includes/global/apply-global-1.html"); ?>
+				<!-- TemplateBeginEditable name="applylocal" -->
+				<?php wdnInclude("/wdn/templates_5.0/includes/local/apply-local.html"); ?>
+				<!-- TemplateEndEditable -->
+				<?php wdnInclude("/wdn/templates_5.0/includes/global/apply-global-2.html"); ?>
+				<?php wdnInclude("/wdn/templates_5.0/includes/global/give-global-1.html"); ?>
+				<!-- TemplateBeginEditable name="givelocal" -->
+				<?php wdnInclude("/wdn/templates_5.0/includes/local/give-local.html"); ?>
+				<!-- TemplateEndEditable -->
+				<?php wdnInclude("/wdn/templates_5.0/includes/global/give-global-2.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/nav-global-2.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/idm.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/search.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/header-global-2.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/logo-lockup-1.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/site-affiliation-1.html"); ?>
+	<!-- TemplateBeginEditable name="affiliation" -->
+	<!-- TemplateEndEditable -->
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/site-affiliation-2.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/site-title-1.html"); ?>
+	<!-- TemplateBeginEditable name="titlegraphic" -->
+	<a class="unl-site-title-medium" href="/">VITA Lincoln</a>
+	<!-- TemplateEndEditable -->
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/site-title-2.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/logo-lockup-2.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/nav-toggle-group.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/nav-menu-1.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/nav-toggle-btn.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/nav-menu-child-1.html"); ?>
```

Comment:
It is kind of unfortunate how many calls now have to be made to fetch all these html files. Do you think this will negatively impact speed too much?

Reply:
I hope not.